### ISSUE TITLE
Campaign monitor subscribe form not working

### ISSUE THREAD

**Issue by GhaziTriki** [forkcms/forkcms#1180]
I have trying subscribe to campaign monitor newsletter with our account but I get the following exception ``` 500 Internal Server Error - CampaignMonitorException ```in line 738  of campaingmonitor.php

The $response in line 593 of campaingmonitor.php is FALSE.

I have tried calling the the API in the browser using our url below and it worked
http://api.createsend.com/api/v3/apikey.json?siteurl=MYURLHERE

By the way the have their own SDK library here https://github.com/campaignmonitor/createsend-php

**Comment by GhaziTriki** [comment]
Another suggestion why not storing the API key in the backend instead of fetching it at every call?

**Comment by WouterSioen** [comment]
I think the API can change from time to time.

**Comment by WouterSioen** [comment]
We're not on version 3 of the API yet. I believe @jeroendesloovere is working on something?

**Comment by jeroendesloovere** [comment]
@WouterSioen I'd love to find some time to work on that shizzle. But currently things are hefty, vacation is coming soon.

**Comment by WouterSioen** [comment]
No problem, take your time ;)

**Comment by Lennerd320i** [comment]
Hi, did you already find some time to check this out?
I noticed that subscribing (at least for me) works in the 3.9.3 version - as in: it pushes the email address in the database and towards Campaign Monitor - but never the less gives the error: something went wrong.
When I enable debugging, the subscribe form passes with the green bar: "ok", it pushes the email address into the database but not towards Campaign Monitor.

So for me the problem should be somewhere in the part when the address is sent over to Campaign Monitor.

Testing it through the backend gives no problems what so ever.

I ran it through google chrome (check element), as well as IE (F12 function), enabled in Symfony the Debugging, but nothing states where the problem is.

Hopefully someone can help me out here, as I should be launching the website this week (in fact today, but hey it's already 2pm :) ) and would like the customers to be able to register for our newsletter.

Tnx
